portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -42,17 +45,20 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     }
 
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-    Query query = new Query("Comment");
+    Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
```

Comment:
A stretch goal would be to allow the user to sort the comments in either ascending or descending order.

Reply:
- [ ] I'll put that on my todo list!